livekit-rtc/livekit/rtc/video_source.py

```diff
@@ -14,27 +14,30 @@
 
 from ._ffi_client import FfiHandle, FfiClient
 from ._proto import ffi_pb2 as proto_ffi
-from ._proto import video_frame_pb2 as proto_video_frame
+from ._proto import video_frame_pb2 as proto_video
 from .video_frame import VideoFrame
 
 
 class VideoSource:
     def __init__(self, width: int, height: int) -> None:
         req = proto_ffi.FfiRequest()
-        req.new_video_source.type = (
-            proto_video_frame.VideoSourceType.VIDEO_SOURCE_NATIVE
-        )
+        req.new_video_source.type = proto_video.VideoSourceType.VIDEO_SOURCE_NATIVE
         req.new_video_source.resolution.width = width
         req.new_video_source.resolution.height = height
 
         resp = FfiClient.instance.request(req)
         self._info = resp.new_video_source.source
         self._ffi_handle = FfiHandle(self._info.handle.id)
 
-    def capture_frame(self, frame: VideoFrame) -> None:
+    def capture_frame(
+        self,
+        frame: VideoFrame,
+        timestamp_us: int = 0,
+        rotation: proto_video.VideoRotation.ValueType = proto_video.VideoRotation.VIDEO_ROTATION_0,
```

Comment:
timestamp and rotation are not encapsulated inside VideoFrame now

Reply:
is timestamp used for a/v sync?